examples/querydbtable_config.nifi.schema.json

```diff
@@ -0,0 +1,245 @@
+                "concurrentlySchedulableTaskCount": 1,
+                "autoTerminatedRelationships": [
+                    "success",
+                    "failure"
+                ],
+                "componentType": "PROCESSOR",
+                "groupIdentifier": "1c2f34d2-ee57-448a-b150-8140f0f7643c"
+            }
+        ],
+        "inputPorts": [],
+        "outputPorts": [],
+        "connections": [
+            {
+                "identifier": "a0bf433a-5ea8-4f5b-87f2-5d060fb368ca",
+                "instanceIdentifier": "ed677727-9dc1-4f9a-82bc-06f2e99b9908",
+                "name": "Query users table/success/Put files to /tmp/output",
+                "position": {
+                    "x": 0.0,
+                    "y": 0.0
+                },
+                "source": {
+                    "id": "0f469727-71d2-420f-aca5-d51599cb7507",
+                    "type": "PROCESSOR",
+                    "groupId": "1c2f34d2-ee57-448a-b150-8140f0f7643c",
+                    "name": "Query users table",
+                    "instanceIdentifier": "9e22e432-9a97-4cd9-a7a4-d1ffeca5fbfe"
+                },
+                "destination": {
+                    "id": "ca0439aa-2454-40f8-90fb-72b792d71bc1",
+                    "type": "PROCESSOR",
+                    "groupId": "1c2f34d2-ee57-448a-b150-8140f0f7643c",
+                    "name": "Put files to /tmp/output",
+                    "instanceIdentifier": "bb8a7729-d7d4-4b82-9cd0-4e5c28a5d52e"
+                },
+                "labelIndex": 1,
+                "zIndex": 0,
+                "selectedRelationships": [
+                    "success"
+                ],
+                "backPressureObjectThreshold": 2000,
+                "backPressureDataSizeThreshold": "100 MB",
+                "flowFileExpiration": "0 seconds",
+                "prioritizers": [],
+                "bends": [],
+                "componentType": "CONNECTION",
+                "groupIdentifier": "1c2f34d2-ee57-448a-b150-8140f0f7643c"
+            }
+        ],
+        "labels": [],
+        "funnels": [],
+        "controllerServices": [
+            {
+                "identifier": "da6e66d2-b185-4ca2-9691-69afb82dfc0b",
+                "instanceIdentifier": "370368b3-75d8-4901-b478-bd039ca7f58a",
+                "name": "ODBCService",
+                "position": {
+                    "x": 0.0,
+                    "y": 0.0
+                },
+                "type": "org.apache.nifi.minifi.sql.controllers.ODBCService",
+                "bundle": {
+                    "group": "org.apache.nifi.minifi",
+                    "artifact": "minifi-sql",
+                    "version": "0.15.0"
+                },
+                "properties": {
+                    "Connection String": "enc{7YHB8DDp3xI5wiKGfsoflVELCXpY7M/v||qKzN8vELnMZGysNDRWitok+XHy0pXkwKF9oR0nNoUet/AfE9aj0oOBHJppSCVKPlq0WjbM82+4HYY3hLcfPssHFn2/zeR1vtDo3wRHptoFmG9pncr8O8noH2tPXHKgqWJnYtwNAiGhEQLuqTSnNXjBe5KSgKNatq}"
```

Comment:
```suggestion
                    "Connection String": "Driver={PostgreSQL ANSI};Server=postgresql-server;Port=5432;Database=postgres;Uid=postgres;Pwd=password;"
```

Reply:
Good point, updated in 3e592636133c986a3557dbec8929929ba3617b04